### DISCUSSION TITLE
Issue with media player

### DISCUSSION THREAD

**Discussion by Renabeth** [lezzthanthree/Needy-Streamer-Overload#15] [Q&A]
I can't get the media player to work no matter what I try. It just stays blank. And I made sure I have it set to the right service (Spotify in this case).

**Comment by lezzthanthree** [comment]
It is actually a current issue on Spotify and Rainmeter... Please check this current [issue](https://github.com/lezzthanthree/Needy-Streamer-Overload/issues/1).
